_includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md

```diff
@@ -0,0 +1,513 @@
+|--------------------------------------------|-------|---------|
+| spy_availability_notification_subscription | table | created |
+
+Make sure that propel entities have been generated successfully by checking their existence. Also, change the generated entity classes to extend from Spryker core classes.
+
+| CLASS PATH                                                                                        | EXTENDS                                                                                                      |
+|---------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| src/Orm/Zed/AvailabilityNotification/Persistence/SpyAvailabilityNotificationSubscription.php      | Spryker\Zed\AvailabilityNotification\Persistence\Propel\AbstractSpyAvailabilityNotificationSubscription      |
+| src/Orm/Zed/AvailabilityNotification/Persistence/SpyAvailabilityNotificationSubscriptionQuery.php | Spryker\Zed\AvailabilityNotification\Persistence\Propel\AbstractSpyAvailabilityNotificationSubscriptionQuery |
+
+Make sure that the following changes have been implemented in transfer objects:
+
+| TRANSFER                                                  | TYPE   | EVENT   | PATH                                                                                   |
+|-----------------------------------------------------------|--------|---------|----------------------------------------------------------------------------------------|
+| AvailabilityNotificationSubscriptionTransfer              | class  | created | src/Generated/Shared/Transfer/AvailabilityNotificationSubscriptionTransfer.php         |
+| AvailabilityNotificationSubscriptionResponseTransfer      | class  | created | src/Generated/Shared/Transfer/AvailabilityNotificationSubscriptionResponseTransfer.php |
+| AvailabilityNotificationSubscriptionRequestTransfer       | class  | created | src/Generated/Shared/Transfer/AvailabilityNotificationSubscriptionRequestTransfer.php  |
+| AvailabilityNotificationSubscriptionMailDataTransfer      | class  | created | src/Generated/Shared/Transfer/AvailabilityNotificationSubscriptionMailDataTransfer.php |
+| AvailabilityNotificationDataTransfer                      | class  | created | src/Generated/Shared/Transfer/AvailabilityNotificationDataTransfer.php                 |
+| CustomerTransfer.availabilityNotificationSubscriptionSkus | column | created | src/Generated/Shared/Transfer/CustomerTransfer.php                                     |
+| MailTransfer.availabilityNotificationSubscriptionMailData | column | created | src/Generated/Shared/Transfer/MailTransfer.php                                         |
+
+{% endinfo_block %}
+
+### 3) Set up behavior
+
+Enable the following behaviors by registering the plugins.
+
+#### Listening to the availability_notification event
```

Comment:
Rewrite the title. It should be a task-based heading and start with a bare infinitive. For example:
```suggestion
#### Configure listening to the availability_notification event
```